runtests.py

```diff
@@ -26,10 +26,14 @@ def runtests(args=None):
     if not any(a for a in args[1:] if not a.startswith("-")):
         args.append("tests/main")
         args.append("mocket")
+        args.append('tests/tests27')
```

Comment:
I removed this directory and moved its tests to `main`, no need for this.

Reply:
Okay, sorry, I am removing it